stratum/hal/bin/barefoot/docker/start-stratum-container.sh

```diff
@@ -11,6 +11,11 @@ DOCKER_IMAGE_TAG=${DOCKER_IMAGE_TAG:-$SDE_VERSION}
 # Try to load the platform string if not already set.
 if [[ -z "$PLATFORM" ]] && [[ -f "/etc/onl/platform" ]]; then
     PLATFORM=$(cat /etc/onl/platform)
+elif [[ -z "$PLATFORM" ]] && [[ -f "/etc/sonic/sonic-environment" ]]; then
+    source /etc/sonic/sonic-environment
+    PLATFORM=$(echo "$PLATFORM" | sed 's/_/-/g')
+    # Disable ONLP when running Stratum on SONiC.
+    set -- ${@} --enable_onlp=false
```

Comment:
This is a little bit cryptic to me.

We might consider explicitly defining:
```
STRATUM_ARGS=${@}
```

Then, just the following here:
```
STRATUM_ARGS+=" --enable_onlp=false"
```

What do you think?

Reply:
Sounds good to me